deploy/kubernetes/base/ds-csi-linode-node.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       hostNetwork: true
       containers:
         - name: csi-node-driver-registrar
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v1.3.0
+          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.12.0
           args:
             - "--v=2"
             - "--csi-address=$(ADDRESS)"
```

Comment:
We should also change `csi-linode-node` to `linode-csi-node` in this file in case we want to keep things consistent.

Reply:
It looks like we'll need to change bunch of other files as well if we want to switch to `linode-*` naming scheme